cmd/ncproxy/ncproxy.go

```diff
@@ -29,6 +29,114 @@ type grpcService struct{}
 
 var _ ncproxygrpc.NetworkConfigProxyServer = &grpcService{}
 
+func (s *grpcService) AssignVF(ctx context.Context, req *ncproxygrpc.AssignVFRequest) (_ *ncproxygrpc.AssignVFResponse, err error) {
```

Comment:
Just to make sure I'm following, the setup is split into two calls. One assigns the device (`AssignVF`), the other does the guest side operations to configure/move the adapter for use by a container (`AddNICVirtualFunction`)?

Reply:
And the opposite for removal

Reply:
yep :)